packaging/linux/flatpak/scripts/additional-install.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+# User Service
+mkdir -p ~/.config/systemd
```

Comment:
This line is not needed since `-p` will make the entire tree if needed. So Next line will create this.